src/neo/Cryptography/Helper.cs

```diff
@@ -75,6 +75,12 @@ public static byte[] RIPEMD160(this byte[] value)
             return ripemd160.ComputeHash(value);
         }
 
+        public static byte[] RIPEMD160(this ReadOnlySpan<byte> value)
+        {
+            byte[] source = value.ToArray();
+            return source.RIPEMD160();
```

Comment:
```suggestion
            byte[] buffer = new byte[32];
            using var ripemd160 = new RIPEMD160Managed();
            ripemd160.TryComputeHash(value, buffer, out _);
            return buffer;
```

Reply:
I have try this way before, but there is exception in UT:

> System.InvalidOperationException
  HResult=0x80131509
  Message=The algorithm's implementation is incorrect.
  Source=System.Security.Cryptography.Primitives
  StackTrace:
   at System.Security.Cryptography.HashAlgorithm.TryHashFinal(Span`1 destination, Int32& bytesWritten)
   at System.Security.Cryptography.HashAlgorithm.TryComputeHash(ReadOnlySpan`1 source, Span`1 destination, Int32& bytesWritten)
   at Neo.Cryptography.Helper.RIPEMD160(ReadOnlySpan`1 value) in D:\project\CrossChain\3.0version\pr\neo\src\neo\Cryptography\Helper.cs:line 84
   at Neo.UnitTests.Cryptography.UT_Cryptography_Helper.TestRIPEMD160() in D:\project\CrossChain\3.0version\pr\neo\tests\neo.UnitTests\Cryptography\UT_Cryptography_Helper.cs:line 140

seems algorithms has issue in RIPMED160Managed TryComputeHash() method

![image](https://user-images.githubusercontent.com/48947753/84367092-5390c800-ac06-11ea-9c16-03fb0db216a9.png)